develop/develop-images/dockerfile_best-practices.md

```diff
@@ -614,10 +614,10 @@ Similar to having constant variables in a program (as opposed to hard-coding
 values), this approach lets you change a single `ENV` instruction to
 auto-magically bump the version of the software in your container.
 
-Each `ENV` line creates a new intermediate layer, just like `RUN` commands. This
-means that even if you unset the environment variable in a future layer, it
-still persists in this layer and its value can be dumped. You can test this by
-creating a Dockerfile like the following, and then building it.
+Each `ENV` line creates a new intermediate image. This means that even if you
+unset the environment variable in a future layer, it still persists in this
+layer and its value can be dumped. You can test this by creating a Dockerfile
+like the following, and then building it.
```

Comment:
There's a mixer of 'layer' and 'image' here now. Would anyone have suggestions on how to make this less ambiguous?

> creates a new intermediate image

> unset the environment variable in a future layer

>  still persists in this layer